src/app/shell/SiteNavigation.module.css

```
@@ -3,52 +3,63 @@
 
 .root a:hover {
-  text-decoration: underline;
+  opacity: 1;
 }
 
-.activeLink {
+.root a.activeLink {
+  opacity: 1;
   font-weight: 600;
-  text-decoration: underline;
+  border-bottom-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)